fabtests/prov/efa/src/multi_ep_mt.c

```diff
@@ -230,9 +238,16 @@ static void *post_sends(void *context)
 			}
 		}
 
-		sleep_time = (rand() % (RANDOM_MAX - RANDOM_MIN + 1)) + RANDOM_MIN;
-		ts.tv_nsec = sleep_time;
-		nanosleep(&ts, NULL);
+		random_sleep();
+
+		if (shared_av)
+			av_idx = 0;
+		else
+			av_idx = idx;
+
+		ret = ft_av_remove(avs[av_idx], 1, &remote_fiaddr[idx], 1, 0);
```

Comment:
I prefer
```
av_idx = shared_av ? 0 : idx;
```
but I don't super care about it.

I also don't think the ft_av_remove call really needs to be in the common code because you're just adding it in one place and it's basically just a pass through of the OFI call. I don't have a strong opinion about it if you have a vision for using it in the future though.
Also you're not checking the return code here so coverity will probably complain

Reply:
Agree.

I have a stronger preference that the `ft_av_remove` call should not be added.  We should call `fi_av_remove` directly and check the return result and print error message accordingly. See the `fi_av_open` call in the same file for an example.